Volume/Networking/API.swift

```diff
@@ -0,0 +1,488 @@
+// @generated
+//  This file was automatically generated and should not be edited.
+
+import Apollo
+import Foundation
+
+public final class GetHomeArticlesQuery: GraphQLQuery {
```

Comment:
there's no sensitive info in this so this isn't urgent, but i believe we don't want this to be committed to github. i believe eatery & uplift both have the API.swift file in their .gitignore